src/meta/src/manager/catalog/mod.rs

```diff
@@ -1857,6 +1862,157 @@ impl CatalogManager {
         Ok(version)
     }
 
+    pub async fn alter_set_schema(
+        &self,
+        object: alter_set_schema_request::Object,
+        new_schema_id: SchemaId,
+    ) -> MetaResult<NotificationVersion> {
+        let core = &mut *self.core.lock().await;
+        let database_core = &mut core.database;
+
+        database_core.ensure_schema_id(new_schema_id)?;
```

Comment:
Check if the schema is unchanged and ignore it if so.

Reply:
If I skip notifying the `Frontend`, which `version` should I return?

Reply:
I think you can use `IGNORED_NOTIFICATION_VERSION` instead.